docs/source/markdown/podman-machine-init.1.md.in

```diff
@@ -44,9 +44,7 @@ Print usage statement.
 Fully qualified path of the ignition file.
 
 If an ignition file is provided, the file
-will be copied into the user's CONF_DIR and renamed.  Additionally, no SSH keys will
-be generated nor will a system connection be made.  It is assumed that the user will
-do these things manually or handle otherwise.
+is copied into the user's CONF_DIR and renamed.  Additionally, no SSH keys are generated nor a system connections made.  It is assumed that the user does these things manually or handled otherwise.
```

Comment:
```suggestion
is copied into the user's CONF_DIR and renamed.  Additionally, no SSH keys are generated, nor are any system connections made.  It is assumed that the user does these things manually or handled otherwise.
```